uart.c

```diff
@@ -48,6 +48,13 @@ uint8_t uart_get_buffered(void) {
     }
     return c;
 }
+
+int uart_getch(void) {

```

Comment:
Implementation incomplete / Is not used anywhere else? | Should be as safer way to get ``uart_get_buffered()``?